payments-api/paymentHandler_test.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPaymentHandler(t *testing.T) {
+	srv := server{router: mux.NewRouter()}
```

Comment:
@firstchild78 likely to go with GRPC, so would need to change internals a bit but for now, lgtm